gamestonk_terminal/comparison_analysis/ca_controller.py

```diff
@@ -9,10 +9,12 @@
 from matplotlib import pyplot as plt
 from gamestonk_terminal import feature_flags as gtff
 from gamestonk_terminal import config_terminal as cfg
+from finvizfinance.screener.overview import Overview
 from gamestonk_terminal.helper_funcs import get_flair, parse_known_args_and_warn
 from gamestonk_terminal.comparison_analysis import yahoo_finance_api as yf_api
 from gamestonk_terminal.comparison_analysis import market_watch_api as mw_api
 from gamestonk_terminal.comparison_analysis import finbrain_api as f_api
+from gamestonk_terminal.comparison_analysis import finviz_view
```

Comment:
If I get rid of this line (import finviz_view) it works

Reply:
This is the most important file of the PR, and I thought I had added it before the commit, but apparently I had not... It should work now (after I make sure there are no errors).

Reply:
Thanks for checking this!